lib/simple_lru.js

```diff
@@ -85,7 +88,7 @@ Cache.prototype.set = function(key,val,hit){
             cacheVal.setValue(val)
         }
 
-        cacheVal = cacheVal ? cacheVal : new ListElement(undefined,undefined,key,val)
+        cacheVal = cacheVal ? cacheVal : new ListElement(undefined,undefined, this.maxTime ? Date.now() : undefined, key, val)
```

Comment:
Need to bump time here on hit: https://github.com/geisbruch/node-simple-lru-cache/pull/3/files#diff-df5bb34be0e9c31fb31e3b40e91f8793R67

This should do it

```
if(hit) {
 this.hit(actual);
 actual.time = this.maxTime ? Date.now() : undefined
}
```